### PR TITLE
Rails 5 compatibility

### PR DESCRIPTION
**Pull request by AnnaKL**

### CODE REVIEW

File: test/lib/superfeedr_engine/engine_test.rb [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+      
+      engine.stub :superclass, true, mock do
+        assert engine.subscribe(instance)
+      end
+      mock.verify
+    end
+
+    test 'unsubscribes feed' do
+      instance = ValidFeed.new
+      mock = MiniTest::Mock.new
```
Comment (shamess) [inline test/lib/superfeedr_engine/engine_test.rb]: We've not added any `mock.expects` so I'm not sure the `mock.verify` is doing anything, is it?

File: test/lib/superfeedr_engine/engine_test.rb [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+module SuperfeedrEngine
+  class EngineTest < ActionController::TestCase
+    def setup
+      @engine = SuperfeedrEngine::Engine
+      @validationError = SuperfeedrEngine::Engine::ValidationError
+    end
+    test 'requires url method on the instance' do
+      instance = Class.new
+      error = assert_raises @validationError do
```
[the remark this answers is not in the document]
Comment (AnnaKL) [reply test/lib/superfeedr_engine/engine_test.rb]: Yeah. I probably should have added rspec but also wanted to learn a bit of minitest :)

File: test/lib/superfeedr_engine/engine_test.rb (hunk 1 above)
Comment (AnnaKL) [reply test/lib/superfeedr_engine/engine_test.rb]: As discussed, those tests have been deleted.

File: lib/superfeedr_engine/engine.rb [hunk 3]
```diff
@@ -5,6 +5,7 @@
 
 module SuperfeedrEngine
   class Engine < ::Rails::Engine
+    class ValidationError < StandardError; end
```
[the remark this answers is not in the document]
Comment (AnnaKL) [reply lib/superfeedr_engine/engine.rb]: Updated 👍

File: test/lib/superfeedr_engine/engine_test.rb [hunk 4]
```diff
@@ -0,0 +1,82 @@
+require 'test_helper'
+require "minitest/autorun"
```
Comment (ascastro) [inline test/lib/superfeedr_engine/engine_test.rb]: Use single quotes

File: test/lib/superfeedr_engine/engine_test.rb [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+module SuperfeedrEngine
+  class EngineTest < ActionController::TestCase
+    def setup
+      @engine = SuperfeedrEngine::Engine
+      @validationError = SuperfeedrEngine::ValidationError
+    end
```
Comment (ascastro) [inline test/lib/superfeedr_engine/engine_test.rb]: Add line break between `def` and `test`

File: lib/superfeedr_engine/version.rb [hunk 6]
```diff
@@ -1,3 +1,3 @@
 module SuperfeedrEngine
-  VERSION = "0.2.0"
+  VERSION = "0.2.1"
```
Comment (ascastro) [inline lib/superfeedr_engine/version.rb]: Since we're changing this, you know what I'm going to say... Yes, single quotes :D

Comment (AnnaKL) [reply lib/superfeedr_engine/version.rb]: hahahah oh yes, sorry about that :) I need to pay more attention to it :)

**Pull request merged**